tests/cluecode/test_finder.py

```diff
@@ -182,6 +182,23 @@ def test_emails_does_filter_junk_domains(self):
         result = find_emails_tester(test_file)
         assert expected == result
 
+    def test_emails_for_ignored_hosts(self):
+        """
+        Add tests for ignored_hosts filter, for email domain hosts.
+        """
```

Comment:
Please avoid using docstrings in tests: either use longer, more ddescriptive test method names or use comments.
Test runner otherwise tend to report the test doc rather than the method name and this does snot help much in reports and failure traces.